autotune/hpo-keras/README.md

```diff
@@ -11,21 +11,30 @@
 
 1. 前往实验管理控制台，在根路径下创建一个名为 tutorial 的文件夹；然后复制该文件夹的 ID。
 
-1. 回到 Notebook 的终端，切换到当前目录下，编辑文件 `autotune.yaml` 或 `autotune_gpu.yaml`（见下），在 `spec.aimd.folder` 字段（第 11 行）粘贴刚才复制的文件夹 ID。
+1. 回到 Notebook 的终端，切换到当前目录下，编辑文件 `autotune.yaml`、`autotune_gpu.yaml` 或 `autotune_gpu_t9ksched.yaml`（见下），在 `spec.aimd.folder` 字段（第 11 行）中粘贴刚才复制的文件夹 ID。
 
     ```shell
     # cd into current directory
     cd ~/tutorial-examples/autotune/hpo-keras
-    vim autotune.yaml
-    # paste API Key as value of `spec.aimd.folder` field (line 11)
+    vim autotune.yaml  # or `autotune_gpu.yaml`, `autotune_gpu_t9ksched.yaml`
+        # paste folder ID (line 11)
     ```
 
-1. 使用 `autotune.yaml`（CPU 训练）或 `autotune_gpu.yaml`（GPU 训练，需要 12 个 GPU）创建 AutoTuneExperiment：
+1. 使用以下 YAML 配置之一创建 AutoTuneExperiment：
+
+    * `autotune.yaml`：使用 CPU 训练
+    * `autotune_gpu.yaml`：使用 GPU 训练（需要 12 个 GPU）
+    * `autotune_gpu_t9ksched.yaml`：使用 GPU 训练（需要 12 个 GPU），并且由 T9k 调度器进行调度（需要编辑文件，在 `spec.trainingConfig.scheduler.t9kScheduler.queue` 字段（第 23 行）中填写队列的名称）
 
     ```shell
     # choose one of the following:
-    kubectl create -f autotune.yaml      # CPU training
-    kubectl create -f autotune_gpu.yaml  # GPU training
+    # 1. CPU training
+    kubectl create -f autotune.yaml
+    # 2. GPU training
+    kubectl create -f autotune_gpu.yaml
+    # 3. GPU training with all Jobs scheduled by T9k scheduler
+    vim autotune_gpu_t9ksched.yaml && kubectl create -f autotune_gpu_t9ksched.yaml
```

Comment:
comment out vim command

```bash
# vim autotune_gpu_t9ksched.yaml
kubectl create -f autotune_gpu_t9ksched.yaml
```

Do this for all other `README.md` files

Reply:
modified